pages/linux/chpasswd.md

```diff
@@ -0,0 +1,20 @@
+# chpasswd
+
+> Change passwords for multiple users via taking input from stdin.
+> More information: <https://manpages.debian.org/jessie/passwd/chpasswd.8.en.html>.
+
+- Change password for a user:
+
+`printf "{{username}}:{{new_password}}" | sudo chpasswd`
+
+- Change passwords for multiple users (Mind the lack of space in the stdin text):
+
+`printf "{{username_one}}:{{new_password_one}}\n{{username_two}}:{{new_password_two}}" | sudo chpasswd`
+
+- Change password into the supplied encrypted password for a user:
```

Comment:
It's helpful to keep the wording similar between examples, so one can quickly spot the differences
```suggestion
- Change the password for a specific user, and specify it in encrypted form:
```